clang/lib/StaticAnalyzer/Checkers/ArrayBoundChecker.cpp

```diff
@@ -1,4 +1,4 @@
-//== ArrayBoundCheckerV2.cpp ------------------------------------*- C++ -*--==//
+//== ArrayBoundChecker.cpp --------------------------------------*- C++ -*--==//
```

Comment:
```suggestion
//== ArrayBoundChecker.cpp -------------------------------------------------==//
```

This isn't a header, we can drop the `-*- C++ -*-` part.